pkg/controller/kibana/config_settings.go

```diff
@@ -8,6 +8,8 @@ import (
 	"context"
 	"path"
 
+	"github.com/elastic/cloud-on-k8s/pkg/utils/net"
```

Comment:
nit: some files have new lines in imports, [not sure if we still care about that](https://github.com/elastic/cloud-on-k8s/blob/master/CONTRIBUTING.md#format-your-code-and-manage-imports).